src/main/java/frc/robot/commands/DefaultDriveCommand.java

```diff
@@ -27,15 +31,26 @@ public DefaultDriveCommand(DrivetrainSubsystem drivetrainSubsystem,
 
     @Override
     public void execute() {
-        // You can use `new ChassisSpeeds(...)` for robot-oriented movement instead of field-oriented movement
-        m_drivetrainSubsystem.drive(
-                ChassisSpeeds.fromFieldRelativeSpeeds(
-                        m_translationXSupplier.getAsDouble(),
-                        m_translationYSupplier.getAsDouble(),
-                        m_rotationSupplier.getAsDouble(),
-                        m_drivetrainSubsystem.getGyroscopeRotation()
-                )
-        );
+        if (m_rotationSupplier.getAsDouble() == 0 && (m_translationXSupplier.getAsDouble() != 0 || m_translationYSupplier.getAsDouble() != 0)) {
```

Comment:
I think this if can be simplified to just check the rotation. It’s ok for the pid to remain on when no joysticks are moving